vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePicker.java

```diff
@@ -1141,5 +1152,26 @@ public DatePickerI18n setCancel(String cancel) {
             this.cancel = cancel;
             return this;
         }
+
+        /**
+         * Gets the {@code referenceDate}.
+         *
+         * @return the reference date
+         */
+        public LocalDate getReferenceDate() {
+            return referenceDate;
+        }
+
+        /**
+         * Sets the {@code referenceDate}.
+         *
+         * @param referenceDate
+         *            the date used to base relative dates on
+         * @return this instance for method chaining
+         */
```

Comment:
Sorry, I missed this in the previous reviews. The API docs here are lacking information, refering to `{@code referenceDate}` is not helpful, and `the date used to base relative dates on` also leaves a lot of room for interpretation.

We should expand the docs a bit to indicate that the reference date is used to determine the century when parsing two-digit years, maybe provide an example, and information on how you can enter a year that actually only has two digits.